feedback-module/src/components/Module.js

```diff
@@ -67,20 +71,17 @@ function Module({ pagetitle, endpoint, dir }) {
     }
   }, [screen]);
 
-  // sendFormData determines which form data state to update, based on the formID
-  const sendFormData = (formID) => {
-    if (formID === "feedback") {
-      updateFeedbackForAPI(processFeedback, [checkedFields, inputQuestions]);
-      requestService("feedback", {
-        id: endpoint,
-        feedback: feedbackForAPI,
-      });
-      console.log(feedbackForAPI);
-    } else if (formID === "research") {
-      trackFutureResearch();
-      updateUserInfo(processUserInfo, [inputQuestions, endpoint]);
-      requestService("userResearch", userInfo);
-      console.log(userInfo);
+  // requestInfo returns the object to be submitted
```

Comment:
This comment is a little vague. Maybe change it to: "requestInfo parses the user's info and returns the object that will be submitted to the Microsoft Flow endpoint".